pkg/http/api/v1/handler.go

```diff
@@ -6,31 +6,27 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+
 	"github.com/anchore/harbor-scanner-adapter/pkg/adapter"
 	"github.com/anchore/harbor-scanner-adapter/pkg/adapter/anchore"
 	"github.com/anchore/harbor-scanner-adapter/pkg/model/harbor"
-	"github.com/gorilla/mux"
-	log "github.com/sirupsen/logrus"
 )
 
 const (
-	AcceptHeader                  = "Accept"
-	ContentTypeHeader             = "Content-Type"
-	AuthorizationHeader           = "Authorization"
-	BearerTokenPrefix             = "bearer"
-	AllMimeTypes                  = "*/*"
-	JSONRequestMimeType           = "application/json"
-	HarborVulnReportv1MimeType    = "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"
-	RawVulnReportMimeType         = "application/vnd.scanner.adapter.vuln.report.raw+json"
-	DockerImageMimeType           = "application/vnd.docker.distribution.manifest.v2+json"
-	OciImageMimeType              = "application/vnd.oci.image.manifest.v1+json"
-	HarborMetadataVulnDbUpdateKey = "harbor.scanner-adapter/vulnerability-database-updated-at"
-	HarborMetadataScannerTypeKey  = "harbor.scanner-adapter/scanner-type"
-	AdapterType                   = "os-package-vulnerability"
-	ScanRequestMimeType           = "application/vnd.scanner.adapter.scan.request+json; version=1.0"
-	ScanResponseMimeType          = "application/vnd.scanner.adapter.scan.response+json; version=1.0"
-	MetadataResponseMimeType      = "application/vnd.scanner.adapter.metadata+json; version=1.0"
-	ErrorResponseMimeType         = "application/vnd.scanner.adapter.error+json; version=1.0"
+	AcceptHeader               = "Accept"
+	ContentTypeHeader          = "Content-Type"
+	AuthorizationHeader        = "Authorization"
+	BearerTokenPrefix          = "bearer"
+	AllMimeTypes               = "*/*"
+	JSONRequestMimeType        = "application/json"
+	HarborVulnReportv1MimeType = "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0"
+	RawVulnReportMimeType      = "application/vnd.scanner.adapter.vuln.report.raw+json"
+	ScanRequestMimeType        = "application/vnd.scanner.adapter.scan.request+json; version=1.0"
+	ScanResponseMimeType       = "application/vnd.scanner.adapter.scan.response+json; version=1.0"
+	MetadataResponseMimeType   = "application/vnd.scanner.adapter.metadata+json; version=1.0"
+	ErrorResponseMimeType      = "application/vnd.scanner.adapter.error+json; version=1.0"
```

Comment:
More got deleted here?

Reply:
Yeah there were a bunch of unused constants so I just removed them to clear up